core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -109,88 +182,110 @@
 
   <ul>
     <li [ngClass]="{'active': currentUrl === '/learn/math'}">
-      <a (click)="toggleClassroomSubmenu()" class="protractor-mobile-test-classroom">
-        <i class="material-icons oppia-sidebar-menu-icon">school</i>
-        <span>{{ 'I18N_SIDEBAR_CLASSROOM' | translate }}</span>
-      </a>
-      <div [hidden]="!classroomSubmenuIsShown" class="classroom-submenu">
```

Comment:
We should still keep the check if the classroom submenu should be shown or not. 'Learn' nav item should only show link to 'community library' incase classroom is not enabled.

Reply:
done!